Codigo/Protocol.bas

```diff
@@ -19422,3 +19433,20 @@ Private Sub HandleDeleteItem(ByVal UserIndex As Integer)
 HandleDeleteItem_Err:
 102     Call TraceError(Err.Number, Err.Description, "Protocol.HandleDeleteItem", Erl)
 End Sub
+
+Private Sub HandleFeatureToggle(ByVal UserIndex As Integer)
+On Error GoTo HandleFeatureToggle_Err:
+    Dim value As Byte
+    Dim name As String
+    value = Reader.ReadInt8
```

Comment:
`value = max(min(MaxValue,Reader.ReadInt8),MinValue)`

clamps the values to valid range